test-resources/infrastructure/public-api.yaml

```diff
@@ -4,6 +4,55 @@ info:
   version: "1.0"
 
 paths:
+  /.well-known/jwks.json:
```

Comment:
Please could we have a happy path integration test for this new endpoint?

Reply:
I didn't think we needed a separate test for it that why I reworked `JwtVerifier` -> `getPublicKeyUsingJwksUri` which calls it.